dataplane/standalone/port.cc

```diff
@@ -41,17 +45,15 @@ sai_status_t Port::create(_In_ uint32_t attr_count,
             break;
           }
         }
-
         break;
     }
   }
-  if (name != "") {
-    grpc::ClientContext context;
-    lemming::dataplane::CreatePortRequest req;
-    lemming::dataplane::CreatePortResponse resp;
-    req.set_id(this->id);
-    req.set_type(forwarding::PORT_TYPE_KERNEL);
-    req.set_kernel_dev(name);
+  grpc::ClientContext context;
+  lemming::dataplane::CreatePortRequest req;
+  lemming::dataplane::CreatePortResponse resp;
+  req.set_id(this->id);
+  if (type == SAI_PORT_TYPE_CPU) {
+    req.set_type(forwarding::PORT_TYPE_CPU_PORT);
```

Comment:
Looks like the if only needs to be on setting the req fields rather than the dataplane call handling? Can we restructure this so that the handling code is not duplicated?